src/main/java/org/folio/rest/impl/PermissionUtils.java

```diff
@@ -0,0 +1,56 @@
+package org.folio.rest.impl;
+
+import org.folio.rest.jaxrs.model.OkapiPermission;
+import org.folio.rest.jaxrs.model.Permission;
+
+public class PermissionUtils {
+
+  private PermissionUtils() {
+    
+  }
+  
+  /**
+   * Compare two objects for equality, w/ checks for null, etc.
+   * 
+   * @param a LHS of comparison
+   * @param b RHS of comparison
+   * @return true if equal, false otherwise
+   */
+  public static boolean equals(Object a, Object b) {
```

Comment:
Drop and use https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Objects.html#equals(java.lang.Object,java.lang.Object)